kbase-extension/static/kbase/js/widgets/narrative_core/upload/stagingAreaViewer.js

```diff
@@ -367,25 +402,33 @@ define([
                     const rowFileName = data[2];
                     //use the name to look up all the data we have
                     let rowFileData = getFileFromName(rowFileName);
+                    
+                    //find the initial singly mapped datatype from the staging service
+                    let suggestedTypes = $(data[5]).find('optgroup[label="Suggested Types"]');
+                    let suggestedType = null;
+                    if (suggestedTypes.children().length == 1) {
+                        var option = suggestedTypes.find('option');
```

Comment:
for any var that won't change (or shouldn't change) you can assign it as a const, others should be let. but i would avoid mixing let + var

Reply:
changed it to const